apps/web-giddh/src/app/home/components/bank-accounts/bank-accounts.component.ts

```
@@ -69,6 +83,19 @@ export class BankAccountsComponent implements OnInit, OnDestroy {
                 this.getAccounts(this.fromDate, this.toDate, 'bankaccounts', null, null, 'true', 20, '', 'closingBalance', 'desc');
             }
         };
+        this.store.pipe(select(profileObj => profileObj.settings.profile), takeUntil(this.destroyed$)).subscribe((profile) => {
+        if(profile?.userEntityRoles){
+            profile.userEntityRoles.forEach(role => {
+                const scopes = role.role.scopes;
+                if (scopes && scopes.some(scope => scope.name === 'INTEGRATION')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)